designs/initial.md

```diff
@@ -0,0 +1,56 @@
+## `@rest` directive
+
+#### arguments:
+- route: path to rest endpoint. This could be a path or a full url. If a path, add to the endpoint given on link creation or from the context
+- provides: a map of variables to url params
```

Comment:
Maybe this can be called "params"?

Reply:
ah yes that makes way more sense!

Reply:
I concur, `provides` is used in other contexts in Apollo, and I "get" that it means the arg is exported from the inner context for client-calling code to pass in, but that feels far from how users coming in for REST params would expect to see things.

Reply:
I concur, `provides` is used in other contexts in Apollo, and I "get" that it means the arg is exported from the inner context for client-calling code to pass in, but that feels far from how users coming in for REST params would expect to see things.